test/core/primitives/CMakeLists.txt

```diff
@@ -8,4 +8,5 @@ addtest(address_test
     )
 target_link_libraries(address_test
     address
+    leb128
```

Comment:
If you use leb128 in address then you should link it to address instead of address test

Reply:
I think the test wouldn't link without that but I can be wrong - will check

Reply:
yes. To make it link you need to link address library with leb128. 
Otherwise, every dependency of address would need to be linked with leb128 manually, whereas it could be avoided by just linking address with lev128